### DISCUSSION TITLE
Alguien sabe como insertar componente dentro de una imágen como fondo?

### DISCUSSION THREAD

**Discussion by iXrst** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#445] [Q&A]
Alguien sabe como hacer lo que aparece como ejemplo en la práctica¿? 
Pero insertando todo un contenido (tablas, containers, div, etc) que se pueda hacer "scroll" dentro de los márgenes de la "pantalla"
![image](https://user-images.githubusercontent.com/14317563/158892674-35ac8500-63a0-47c8-8e94-3c40699c3330.png)

**Comment by doguedogue** [comment]
Pones una imagen de fondo y luego agregas tus datos arriba, la clave (position/relative/fixed)

**Reply by carlos-gmx3** [reply]
Puedes hacerlo con position relative, absolute y haciendo uso de z-index
O a traves de estilos a un contanedor padre darle el estilo de backgroung-image
En cualquiera de los 2 casos necesitar alinear el div de los datos y la imagen del Poke
<div class="dex">
<div class="pokeImg"></div>
</div>

**Comment by Ingrzs** [comment]
Background image al div en css

El div padre debe tener 3 div hijos 
Ya tu iras calculando el tamaño

Divpadre{
Divhijo {
Dibisnieto{
Divtatanieto{}
   }
 }
}

Claro yo lo puse con parentesis  para el ejemplo

**Comment by hackit92** [comment]
Ponle esta propiedad CSS a tu contenedor

<code> overflow-y: scroll;</code>
